src/Abp.Zero/Authorization/Users/AbpUserManager.cs

```diff
@@ -526,6 +526,7 @@ public virtual async Task SetOrganizationUnitsAsync(TUser user, params long[] or
                 }
             }
 
+            await _unitOfWorkManager.Current.SaveChangesAsync();
```

Comment:
hi @demirmusa 

The SetOrganizationUnitsAsync method requires UnitOfWork, otherwise the unit test will fail.